config/rbac/role.yaml

```diff
@@ -14,6 +14,18 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - create
```

Comment:
I think we only need create/get/list if we install on start. Or list/watch/create if we want to reconcile and reinstall.